Make this build with musl, uid_t/pid_t are defined in <unistd.h>.

--- a/utils/proc-llist.h	2014-01-11 11:38:42.960868985 +0100
+++ b/utils/proc-llist.h	2014-01-11 11:39:01.776988967 +0100
@@ -24,6 +24,7 @@
 #ifndef PROC_HEADER
 #define PROC_HEADER
 
+#include <unistd.h> /* uid_t, pid_t */
 #include "config.h"
 
 
